src/main/resources/data/alaskanativecraft/recipes/purple_dye_from_cloudberries.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "minecraft:crafting_shapeless",
+  "group": "purple_dye",
+  "ingredients": [
+    {
+      "item": "alaskanativecraft:cloudberries"
+    }
+  ],
+  "result": {
+    "item": "minecraft:purple_dye"
+  }
+}
```

Comment:
Purple dye from cloudberries? They're a rich yellow-gold color, or at worst a red

Reply:
Researched this a bit, dye from them tends to be purple

Reply:
Same deal with salmonberries

Reply:
https://eflora.neocities.org/Rubus%20spectabilis.html For my source on having the salmonberry dye be purple

Reply:
As for cloudberry dye... I'm actually having a hard time finding info on that.

Reply:
And on the note of berries, I nearly added lingonberries and cranberries

Reply:
Found a source claiming they can also be made into a purple-blue dye

Reply:
I came across that same source cause I did a sanity check when I saw these recipes. I still think that even creative liberties in making them produce dyes close to the berry color makes more sense. I'm pretty sure the purple dye referenced is from overripe berries, or perhaps the leaves of the plant.